crowdfund/contracts/SmartPoolToken.sol

```diff
@@ -0,0 +1,131 @@
+pragma solidity ^0.4.0;
+
+import "./ERC20.sol";
+import "./Lockable.sol";
+
+contract SmartPoolToken is ERC20, Lockable {
+    address public theTeam;
+    mapping(address => uint) public _tokenBalances;
+    mapping(address => uint) public _donationBalances;
+    mapping(uint => address) public _donors;
```

Comment:
Since `_donors` maps an index to donor's address, it's a way for us to iterate through donors to get their address, combine with `_tokenBalances` and `_donationBalances` we can also get donation and token amount too. I think it would be helpful for us in future?